<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>752</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_3DCamera">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>3D相机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2DCamera1">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>SimSun-ExtB</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>2D相机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_config">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_detectstart">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>开始检测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_detectstop">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>停止检测</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:lightgrey;
</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page_3D">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3d">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:white;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:white;

</string>
            </property>
            <property name="title">
             <string>3D相机控制</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_init3">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>初始化</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_grab3">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>触发拍照</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_save3">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>保存图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_close3">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>关闭相机</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_testSDK">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2D1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="CammerWidget" name="widget_Camera2D_1" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:white;
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_paramter">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>100</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>行:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>100</y>
            <width>68</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>行间距:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>150</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>列:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>150</y>
            <width>68</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>列间距:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_rowNum">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>92</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_ColNum">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>140</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_rowGap">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>90</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_ColGap">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>140</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setRow">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>90</y>
            <width>84</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string>写入</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_gray_maximum">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>200</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setGray">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>200</y>
            <width>84</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>写入</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_gray_minimum">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>200</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>277</x>
            <y>199</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>灰度值上限:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>199</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>灰度值下限：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_pr1Step">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>261</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setPr1">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>260</y>
            <width>84</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>触发</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>PR1运行</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setPr0">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>320</y>
            <width>131</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>写入数据</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>320</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>PR0回零</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>370</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>PR0回零</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setJOG">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>370</y>
            <width>84</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>触发</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setPr0_2">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>320</y>
            <width>131</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>归零</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_JOGpositive">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>430</y>
            <width>84</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>JOG</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_setJOGnagetive">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>430</y>
            <width>84</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>JOG-</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="logtextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:lightgrey;
</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14.4pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="cursorWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CammerWidget</class>
   <extends>QWidget</extends>
   <header>CammerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
